core/payment-driver/zksync/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "ya-zksync-driver"
+version = "0.1.0"
+authors = ["Golem Factory <contact@golem.network>"]
+edition = "2018"
+
+[features]
+default = []
+
+[dependencies]
+actix = "0.9"
+anyhow = "1.0"
+num = { version = "0.2", features = ["serde"] }
+bigdecimal = { version = "0.1.0" }
+client = { git = "https://github.com/golemfactory/zksync", branch = "mwu/decouple-client-server"}
+# client = { path = "../../../../zksync/core/client" }
```

Comment:
I do not like leaving comments as this one.

Reply:
Good one, removed :)